extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-init-job.yaml

```diff
@@ -0,0 +1,45 @@
+{{ if .Values.webhook.enable }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-init
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "50"
+spec:
+  template:
+    metadata:
+      name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-init
+    spec:
+      serviceAccountName: {{ .Values.webhook.serviceAccount.name }}
+      restartPolicy: OnFailure
+      {{- if .Values.webhook.certSecret.create }}
+      initContainers:
+      - name: create-certs
+        image: {{ index .Values "aissemble-quarkus-chart" "deployment" "image" "repo" }}{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "name" }}:{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "tag" | default .Chart.AppVersion }}
+        imagePullPolicy: {{ index .Values "aissemble-quarkus-chart" "deployment" "image" "imagePullPolicy"}}
+        command: ["/usr/bin/genCerts.sh"]
+        args: [
+            "--namespace", "{{ .Release.Namespace }}",
+            "--service", "{{ index .Values "aissemble-quarkus-chart" "app" "name" }}",
+            "--secret-name", "{{ .Values.webhook.certSecret.name }}"
+          ]
+      {{- end }}
+      containers:
+      - name: create-webhook
+        image: {{ index .Values "aissemble-quarkus-chart" "deployment" "image" "repo" }}{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "name" }}:{{ index .Values "aissemble-quarkus-chart" "deployment" "image" "tag" | default .Chart.AppVersion }}
+        imagePullPolicy: {{ index .Values "aissemble-quarkus-chart" "deployment" "image" "imagePullPolicy"}}
+        command: ["/usr/bin/genWebhook.sh"]
+        args: [
+            "--ca-bundle", "$(CA_BUNDLE)",
```

Comment:
Q: Why create a subshell here? Couldn't we just do `$CA_BUNDLE`?

Reply:
Not sure why, but when I do just `$CA_BUNDLE` only the last line of the variable gets passed in. Surrounding it with () ensure the full value gets passed in.